pkgs/development/python-modules/mizani/default.nix

```diff
@@ -1,15 +1,20 @@
-{ buildPythonPackage
+{ lib
+, buildPythonPackage
 , fetchFromGitHub
-, lib
 , matplotlib
 , palettable
 , pandas
 , pytestCheckHook
+, pythonOlder
+, scipy
 }:
 
 buildPythonPackage rec {
   pname = "mizani";
   version = "0.7.3";
+  format = "setuptools";
```

Comment:
Why is this required now? This wasn't necessary when I originally packaged it, and it doesn't look like `src` has changed.

Reply:
The way how Python packages are build will change in the near future. For now it's more of a "marker" and I add them when I have to edit the files.